credsweeper/app.py

```diff
@@ -69,38 +70,74 @@ def __init__(self,
             ml_threshold: float or string value to specify threshold for the ml model
             find_by_ext: boolean - files will be reported by extension
             depth: int - how deep container files will be scanned
+            doc: boolean - document-specific scanning
             size_limit: optional string integer or human-readable format to skip oversize files
             exclude_lines: lines to omit in scan. Will be added to the lines already in config
             exclude_values: values to omit in scan. Will be added to the values already in config
 
         """
         self.pool_count: int = int(pool_count) if int(pool_count) > 1 else 1
+        config_dict = self._get_config_dict(config_path, api_validation, use_filters, find_by_ext, depth, doc,
+                                            size_limit, exclude_lines, exclude_values)
+        doc_config_dict = self._get_config_dict(None, api_validation, use_filters, find_by_ext, depth, doc, size_limit,
+                                                exclude_lines, exclude_values, "doc_config.json")
+        self.config = Config(config_dict)
+        self.doc_config = Config(doc_config_dict)
+        self.scanner = Scanner(self.config, rule_path)
+        self.doc_scanner = Scanner(self.config, self._get_doc_rule_path())
+        self.deep_scanner = DeepScanner(self.config, self.scanner)
+        self.deep_doc_scanner = DeepScanner(self.doc_config, self.doc_scanner)
```

Comment:
```suggestion
        self.doc_scanner = DeepScanner(self.doc_config, self.doc_scanner)
```

Reply:
Actually there is `self.doc_scanner` above. :cry: